libraries/networking/src/DomainHandler.cpp

```diff
@@ -233,21 +235,15 @@ void DomainHandler::requestDomainSettings() {
         emit settingsReceived(_settingsObject);
     } else {
         if (_settingsObject.isEmpty()) {
-            // setup the URL required to grab settings JSON
-            QUrl settingsJSONURL;
-            settingsJSONURL.setScheme("http");
-            settingsJSONURL.setHost(_hostname);
-            settingsJSONURL.setPort(DOMAIN_SERVER_HTTP_PORT);
-            settingsJSONURL.setPath("/settings.json");
+            qCDebug(networking) << "Requesting settings from domain server";
+
             Assignment::Type assignmentType = Assignment::typeForNodeType(DependencyManager::get<NodeList>()->getOwnerType());
-            settingsJSONURL.setQuery(QString("type=%1").arg(assignmentType));
 
-            qCDebug(networking) << "Requesting domain-server settings at" << settingsJSONURL.toString();
+            auto packet = NLPacket::create(PacketType::DomainSettingsRequest, sizeof(assignmentType), true, false);
+            packet->writePrimitive(assignmentType);
 
-            QNetworkRequest settingsRequest(settingsJSONURL);
-            settingsRequest.setHeader(QNetworkRequest::UserAgentHeader, HIGH_FIDELITY_USER_AGENT);
-            QNetworkReply* reply = NetworkAccessManager::getInstance().get(settingsRequest);
-            connect(reply, &QNetworkReply::finished, this, &DomainHandler::settingsRequestFinished);
+            auto nodeList = DependencyManager::get<LimitedNodeList>();
+            nodeList->sendPacket(std::move(packet), _sockAddr);
```

Comment:
I did this deliberately - I could go either way, but I generally prefer limiting the number of moving parts on a single line.